net/miniupnpd/files/upnpd.config

```diff
@@ -0,0 +1,28 @@
+config upnpd config
+	option enabled		0
+	option enable_natpmp	1
+	option enable_upnp	1
+	option secure_mode	1
+	option log_output	0
+	option download 	1024
+	option upload   	512
+        #by default, looked up dynamically from ubus
+	#option external_iface	wan
+	option internal_iface	lan
+	option port		5000
+	option upnp_lease_file	/var/upnp.leases
+	option force_igd_desc_v1 0
+	
+config perm_rule
+	option action		allow
+	option ext_ports	1024-65535
+	option int_addr		0.0.0.0/0	# Does not override secure_mode
+	option int_ports	1024-65535
+	option comment		"Allow high ports"
```

Comment:
Why do you need `option comment ...`?  What's wrong with `# ....` instead?

Reply:
> Why do you need option comment ...? What's wrong with # .... instead?

To be visible and usable in the actual user interface...

![image](https://user-images.githubusercontent.com/7926856/39469023-bbe317d8-4d3e-11e8-9503-a77686c05fe5.png)

Reply:
Okay, that's what I thought but I didn't see anywhere that the comment was being squirted into a file...